graspologic/datasets/base.py

```diff
@@ -95,3 +101,75 @@ def load_drosophila_right(return_labels=False):
         return graph, labels
     else:
         return graph
+
+
+def load_mice():
+    """
+    Load connectomes of mice from distinct genotypes.
+
+    Dataset of 32 mouse connectomes derived from whole-brain diffusion
+    magnetic resonance imaging of four distinct mouse genotypes:
+    BTBR T+ Itpr3tf/J (BTBR), C57BL/6J(B6), CAST/EiJ (CAST), and DBA/2J (DBA2).
+    For each strain, connectomes were generated from eight age-matched mice
+    (N = 8 per strain), with a sex distribution of four males and four females.
+    Each connectome was parcellated using asymmetric Waxholm Space, yielding a
+    vertex set with a total of 332 regions of interest (ROIs) symmetrically
+    distributed across the left and right hemispheres. Within a given
+    hemisphere, there are seven superstructures consisting up multiple ROIs,
+    resulting in a total of 14 distinct communities ineach connectome.
+
+    Returns
+    -------
+    data : :class:`~sklearn.utils.Bunch`
+        Dictionary-like object, with the following attributes.
+
+        graphs : list of np.ndarray
+            List of adjacency matrices of the connectome
+        labels : np.ndarray
+            Array of string labels for each mouse (subject)
+        atlas : pd.DataFrame
+            DataFrame of information for each ROI
+        blocks : pd.DataFrame
+            DataFrame of block assignments for each ROI
+        features : pd.DataFrame
+            DataFrame of anatomical features for each ROI in each connectome
+
+    References
+    ----------
+    .. [1] Wang, N., Anderson, R. J., Ashbrook, D. G., Gopalakrishnan, V.,
+           Park, Y., Priebe, C. E., ... & Johnson, G. A. (2020). Variability
+           and heritability of mouse brain structure: Microscopic MRI atlases
+           and connectomes for diverse strains. NeuroImage.
+           https://doi.org/10.1016/j.neuroimage.2020.117274
+    """
+
+    data = Path(__file__).parent.joinpath("mice")
+
+    # Load all connectomes
+    graphs = import_edgelist(data.joinpath("edgelists"))
+
+    # Read the participants file and get genotype labels
+    participants = pd.read_csv(data.joinpath("participants.csv"))
+    labels = participants["genotype"].values
+
+    # Read the atlas and block information
+    atlas = pd.read_csv(data.joinpath("atlas.csv"))
+    blocks = pd.read_csv(data.joinpath("blocks.csv"))
+
+    # Read features
+    tmp = []
+    for fl in data.joinpath("features").glob("*" + "csv"):
+        subid = fl.stem
+        df = pd.read_csv(fl, skiprows=2)
+        df["participant_id"] = subid
+        tmp.append(df)
+    features = pd.concat(tmp, axis=0)
+    features = features.reset_index(drop=True)
+
+    return Bunch(
```

Comment:
this is the first time I've seen a Bunch in the wild

Reply:
super useful for holding the multiple sources of data this method returns

also prefer being able to call specific data as an attribute, rather than having to use a dictionary key setup